packages/integration-tests/src/tests/api/hooks.test.ts

```diff
@@ -8,13 +8,19 @@ import { enableAllPasswordSignInMethods } from '#src/helpers/sign-in-experience.
 import { generateNewUser, generateNewUserProfile } from '#src/helpers/user.js';
 import { waitFor } from '#src/utils.js';
 
-const createPayload = (event: HookEvent, url = 'not_work_url'): Partial<Hook> => ({
-  event,
+// TODO: @xiaoyijun update this type in the hook iteration phase 2 (LOG-6082)
```

Comment:
this TODO is not necessary since you already created an issue?

Reply:
Yeah, updated.